docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,42 +1,60 @@
 ### Workflow
 
-#### Contributions
+#### Gatekeepers
 
-All code changes and additions to Mbed OS are handled through GitHub. If you want to contribute, either by adding features or by fixing bugs, please follow the guidelines for [new features](#contributing-new-features-to-mbed-os) and [bugs](#reporting-and-fixing-bugs). In both cases, please follow the [code style guide and GitHub pull request guidelines](/docs/v5.7/reference/guidelines.html#style)</a>. Please also read the [CLA](/docs/v5.7/reference/guidelines.html#cla) guidelines because we will immediately close pull requests submitted without a CLA.
+##### Mbed OS maintainers
 
-#### Contributing new features to Mbed OS
+The maintainers are a small group of Mbed OS engineers who are responsible for the Mbed OS codebase. Their primary role is to progress contributions, both internal and external, from the initial pull request state through to released code. 
 
-Before contributing an enhancement (new feature, new port and so on), please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on a related feature.
+Responsibilities:
 
-Patch contributions can only be accepted through GitHub by creating a pull request from forked versions of our repositories. This allows us to review the contributions in a user friendly and reliable way, under public scrutiny.
+1. Check for CLA (Contributor License Agreement) compliance.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
 
-Please create separate pull requests for each concern; each pull request should have a clear unity of purpose. In particular, separate code formatting and style changes from functional changes. This makes each patch’s true contribution clearer and therefore quicker and easier to review.
+The current maintainers are:
 
-#### Reporting and fixing bugs
+* Anna Bridge [adbridge](https://os.mbed.com/users/AnnaBridge).
```

Comment:
Query: If we're linking to the os.mbed.com profile, does it make sense to use the GH username?

Reply:
Good point. I think we should either link to the GH user page, or use the os.mbed.com profile username.

I'm personally more of a fan of the former.

Reply:
The reason I left github name is that pull requests are via github so that name is visible to contributors, however name might be sufficient and then link to mbed OS?